uktrade/data-hub-frontend
# CLS2-1013: Show when investment project originates from EYB lead

## Description of change

This PR addresses [CLS2-1013](https://uktrade.atlassian.net/browse/CLS2-1013)

- It adds a static `Generated from: EYB leads` item to the end of the meta list when a `InvestmentProject` originates from a `EYB Lead`
- Adds new tests for the untested meta list component

## Test instructions

- Create a test EYB lead and assign it to an investment project
- Head over to the investment project details page
- Just above the project's timeline, the meta list will be present. A new item should appear as per screenshot

Note: this is all inlined, so it will get automatically pushed to a new line at page break

## Screenshots

![Screenshot 2024-11-08 at 11 46 07](https://github.com/user-attachments/assets/051e2769-228f-4363-9039-b6ba873b5e0d)


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'



[CLS2-1013]: https://uktrade.atlassian.net/browse/CLS2-1013?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
